Tests/SpelledPitchTests/PitchSpellerTests.swift

```diff
@@ -264,4 +264,48 @@ class PitchSpellerTests: XCTestCase {
         ]
         XCTAssertEqual(result, expected)
     }
+
+    func testSpelledOneTwoOverDNatural() {
+        let pitches: [Int: Pitch] = [0: 61, 1: 62]
+        let pitchSpeller = PitchSpeller(pitches: pitches, parsimonyPivot: Pitch.Spelling(.d))
+        let result = pitchSpeller.spell()
+        let expected: [Int: SpelledPitch] = [
+            0: SpelledPitch(Pitch.Spelling(.c, .sharp), 4),
+            1: SpelledPitch(Pitch.Spelling(.d), 4)
+        ]
+        XCTAssertEqual(result, expected)
+    }
+
+    func testSpelledOneThreeOverDNatural() {
+        let pitches: [Int: Pitch] = [0: 61, 1: 63]
+        let pitchSpeller = PitchSpeller(pitches: pitches, parsimonyPivot: Pitch.Spelling(.d))
+        let result = pitchSpeller.spell()
+        let expected: [Int: SpelledPitch] = [
+            0: SpelledPitch(Pitch.Spelling(.c, .sharp), 4),
+            1: SpelledPitch(Pitch.Spelling(.d, .sharp), 4)
+        ]
+        XCTAssertEqual(result, expected)
```

Comment:
I would also consider `[0: SpelledPitch(Pitch.Spelling(.d, .flat), 4), 1: SpelledPitch(Pitch.Spelling(.e, .flat), 4)]` to be a valid spelling over `.d` ... ? Can we instead check for predicates like `modifiersInSameDirection` and `noDoubles` / ("parsimony condition") ?

Reply:
These predicates could also be helpful for automatic test generation, since they should be true in every dyad and simpler cases with more pitches. What do you think, @jsbean? Where would they live ?

Reply:
I like the `modifiersInSameDirection`, and is in fact similar to a construct that was in place before.

However, is `modifiersInSameDirection` more an expression of "interval optimization" rather than "notational parsimony"? We have a lot of `IntervalDescriptor` machinery that can be very useful for this.

Could the "no doubles" condition be slightly more-generally represented as average line-of-fifths distance?

Reply:
> These predicates could also be helpful for automatic test generation, since they should be true in every dyad and simpler cases with more pitches. What do you think, @jsbean? Where would they live ?

That's a great question. We could create a `Fixtures` directory in the `SpelledPitchTests`. We could start there, and be scrappy with it. Then, if we think these operations would be more widely applicable, we can wash 'em off and move them upstream.